packages/components/Button/src/ToggleButton/ToggleButton.tsx

```diff
@@ -27,9 +27,10 @@ export const ToggleButton = compose<ToggleButtonType>({
   useRender: (userProps: ToggleButtonProps, useSlots: UseSlots<ToggleButtonType>) => {
     const iconProps = createIconProps(userProps.icon);
     const toggleButton = useToggleButton(userProps);
+    const theme = useFluentTheme();
 
     // grab the styled slots
-    const Slots = useSlots(userProps, (layer) => buttonLookup(layer, toggleButton.state, userProps));
+    const Slots = useSlots(userProps, (layer) => buttonLookup(layer, toggleButton.state, userProps, theme));
```

Comment:
Huh. I'm surprised this needed to be manually added.

Reply:
Surprised that it wasn't already there.. or surprised / questioning the need for the change?